jenkins/deploy/wf/primary/main/config-default/config-default.template

```diff
@@ -0,0 +1,219 @@
+            <value>4</value>
+        </property>
+
+	<!-- requested in:
+	https://github.com/openaire/iis/issues/932
+	 -->
+        <property>
+            <name>citationmatchingFuzzySparkExecutorMemory</name>
+            <value>10G</value>
+        </property>
+        <property>
+            <name>citationmatchingFuzzySparkExecutorOverhead</name>
+            <value>4096</value>
+        </property>
+
+	<property>
+            <name>input_softwareheritage_origins_db</name>
+            <value>/cache/external-resources/software-heritage/origins.db</value>
+        </property>
+
+	<!-- required after integrating git#888 -->
+	<property>
+            <name>affiliationmatchingProjectFundingclassWhitelistRegex</name>
+            <value>^(EC|NSF)::.*</value>
+        </property>
+
+<property>
+        <name>import_content_objectstore_s3_endpoint</name>
+        <value>s3.icm.edu.pl</value>
+</property>
+
+        <property>
+            <name>import_patent_json</name>
+            <value>/cache/external-resources/patents/2019-06-12/patents_epo.txt</value>
+        </property>
+
+<!-- required by spark1 module after configuring spark2 as default one on IIS cluster -->
+<property>
+    <name>oozieActionShareLibForSpark1</name>
+    <value>spark</value>
+</property>
+<property>
+    <name>spark1YarnHistoryServerAddress</name>
+    <value>http://iis-cdh5-test-gw.ocean.icm.edu.pl:18088</value>
+</property>
+<property>
+    <name>spark1EventLogDir</name>
+    <value>/user/spark/applicationHistory</value>
+</property>	
+
+<!-- spark2 related properties are required only until switching the default cluster config to spark2 -->
+<property>
+    <name>oozieActionShareLibForSpark2</name>
+    <value>spark2</value>
+</property>
+<property>
+    <name>spark2YarnHistoryServerAddress</name>
+    <value>http://iis-cdh5-test-gw.ocean.icm.edu.pl:18089</value>
+</property>
+<property>
+    <name>spark2EventLogDir</name>
+    <value>/user/spark/spark2ApplicationHistory</value>
+</property>
+
+<!-- git#991: Prevent mining modules from occupying cluster resources for too long -->
+<property>
+    <name>documentsclassificationNumberOfPartitions</name>
+    <value>10000</value>
+</property>
+
+<!-- git#1051 related to the new InfoSpace graph persistance format (set to parquet by default) -->
+<property>
+    <name>input_format</name>
+    <value>json</value>
+</property>
+
+<!-- git#1000: Export IIS metrics to prometheus pushgateway -->
+<property>
+    <name>metric_pusher_address</name>
+    <value>prometheus.openaire.eu:9091</value>
+</property>
+
+<!-- git#1070: EPO endpoint based patent mining -->
+<property>
+    <name>export_patent_date_of_collection</name>
+    <value>2020-04-27T16:47</value>
+</property>
+<property>
+    <name>patentServiceAuthnConsumerKey</name>
+    <value>fzkakRNW3punGAstva3jyN2uffw91tn2</value>
+</property>
+<property>
+    <name>patentServiceAuthnConsumerSecret</name>
+    <value>CIBLZeO34hCK6rwx</value>
```

Comment:
OK, this could definitely be a reason for keeping such information in a private project. I forgot we define secrets in those templates. S3 cluster credentials are stored in a keystore so only the path is defined within the template but for a newly introduced/refactored patents this is not the case.

This may be a deal-breaker for the deployment scripts location. Even when we find a way to hide patents EPO credentials it may happen someone will define other sensitive information in future.

Reply:
As discussed during daily call after approving this PR I will move back the final version of the code to `iis-deployment` private repo.